CHANGELOG-zh-CN.md

```diff
@@ -1,5 +1,171 @@
 # OceanBase Developer Center (ODC) CHANGELOG
 
+## 4.2.3 (2023-12-21)
+
+### 功能变化
+
+数据源
+
+- 允许数据源绑定到项目
+- 支持 OceanBase Sharding MySQL 数据源
+- 支持克隆数据源
+
+导入导出
+
+- 支持原生 MySQL 数据源的导入导出
+- OceanBase 数据源的导入导出任务配置页面中不再提供 SYS 账户配置
+
+数据库对象管理
+
+- OceanBase MySQL 以及原生 MySQL 模式下支持 GIS 数据类型
+- 白屏创建或删除索引时给出高风险操作提示
+
+基于项目的管控协同
+
+- 增加 2 个内置项目角色：安全管理员，参与者；安全管理员被允许管理项目的敏感列和参与审批，参与者被允许参与审批
+- 允许用户申请项目权限
+- 禁止删除以 DBA 或 项目 OWNER 角色归属到任意项目中的用户
+
+SQL-Check
+
+- 增加了问题定位功能，允许用户将违反的安全规则和具体的 SQL 联系在一起
+
+数据库连接会话
+
+- 增加了自动重连机制，避免长时间不使用场景下会话销毁导致的报错及易用性问题
+
+分区计划
+
+- 支持定时调度
+
+SQL 执行
+
+- SQL 片段最大支持 65535 大小的内容
+- 支持 Ctrl+Enter 快捷键执行当前语句
+
+堡垒机集成
+
+- 支持 SQL Check
+
+DLM
+
+- 支持日志查看
+
```

Comment:
This version of the FullLinkTrace has undergone these optimizations:
1. Able to output JSON files supported by Jaeger
2. Optimized visual effects
3. Added span list view, providing search and sorting functions for span